doc/source/commands.rst

```diff
@@ -1,9 +1,9 @@
-KIWI Commands
-=============
+Command Line
+============
 
 .. note::
 
-   This document provides a list of the existing KIWI commands
+   This document provides a list of the existing {kiwi} commands
```

Comment:
As this will be a manpage at some point, I would add a short explanation about our acronym "NG", using the "full" name with the `{kiwi-full}` placeholder:

```suggestion
   This document provides a list of the existing {kiwi-full} ({kiwi}) commands
```

Reply:
yes done and also updated the man pages. kiwi-ng is the actual name for the binary. Other names (kiw, kiwi-ng-3) are symlinks that can be used too. But the man pages should use the main name